File: Free_ChatGPT/sistemaDeTipos/tipos.py
def promedio_mayores_que(precios, umbral):
    suma = 0
    cantidad = 0

    for precio in precios:
        if precio > umbral:
            suma += precio
            cantidad += 1

    return suma / cantidad if cantidad > 0 else 0

# Ejemplo de uso
precios = [10, 20, 30, 40, 50]
promedio = promedio_mayores_que(precios, 25)

print(promedio)  # 40.0

File: Free_ChatGPT/sistemaDeTipos/tipos.js
"use strict";

function promedioMayoresQue(precios, umbral) {
    let suma = 0;
    let cantidad = 0;

    for (let i = 0; i < precios.length; i++) {
        const precio = precios[i];

        if (precio > umbral) {
            suma += precio;
            cantidad++;
        }
    }

    return cantidad === 0 ? 0 : suma / cantidad;
}

const precios = ["10", "20", "30", "40"];

console.log(promedioMayoresQue(precios, 25)); // 35
console.log(promedioMayoresQue(precios, 50)); // 0
console.log(precios); // [10, 20, 30, 40] (sin modificaciones)